parquet-column/src/main/java/org/apache/parquet/schema/Type.java

```diff
@@ -362,5 +362,4 @@ void checkContains(Type subType) {
    * @return the converted tree
    */
    abstract <T> T convert(List<GroupType> path, TypeConverter<T> converter);
-
```

Comment:
nit: Please undo formatting changes in this file.

Reply:
fixed